market/connector/tel-search-connector/meta.json

```diff
@@ -0,0 +1,76 @@
+{
+	"id": "tel-search-ch-connector",
+	"name": "tel.search.ch",
+	"description": "It has never been easier to retrieve phone directory entries via the tel.search.ch API.",
```

Comment:
Retrieve phone directory entries from Switzerland within your process work using the tel.search.ch API.